src/main/java/net/minecraftforge/fml/common/asm/transformers/EventSubscriberTransformer.java

```diff
@@ -34,6 +34,10 @@
 import com.google.common.base.Predicate;
 import com.google.common.collect.Iterables;
 
+/**
+ * @deprecated this transformer is deprecated. See {@link net.minecraftforge.fml.common.eventhandler.EventListenerFactory#LOOKUP}
```

Comment:
This deprecation info contains too few information(, basically saying nothing).
```suggestion
 * @deprecated Forcing public access is no longer required thanks to new Lookup with trusted member access, see {@link net.minecraftforge.fml.common.eventhandler.EventListenerFactory#LOOKUP}
```